### ISSUE TITLE
Instructions on troubleshooting

### ISSUE THREAD

**Issue by matueranet** [evancohen/sonus#37]
I am attempting to build a library with this project. The hotword detection with the snowboy model works. But I never receive a transcribe event after that. How can I debug/troubleshoot what is going wrong. Any suggestions?

**Comment by evancohen** [comment]
Yeah, having some troubleshooting docs is probably be a good idea. 90% of the time the symptoms you describe are caused by something wrong with the billing configuration.

I have instructions for how to correctly enable billing for another project which should be the same for Sonus: https://docs.smart-mirror.io/docs/configuring_voice.html#setting-up-speech-recognition

**Comment by matueranet** [comment]
I checked an billing is enabled for the project. Next step I guess is to start with `node --inspect` and step through the code.

**Comment by jaumard** [comment]
I'm trying to implement the example, I also have the hot key detection working but not the google speech part :( 
I check on my google cloud account and I see there some calls but all in 403 errors :( I create new json file and project ID match so I'm out of idea of what's going wrong.

Any ideas ?

**Comment by evancohen** [comment]
@jaumard 403 most likely means there's a problem with your billing information. I have some instructions on how to correctly enable it do that for another project: [Setting up speech recognitinon](https://docs.smart-mirror.io/docs/configuring_voice.html#setting-up-speech-recognition).

**Comment by jaumard** [comment]
@evancohen did the trick for me ! Thanks ! All working for me :) any way to choose the langage ? For now it's english but what if I want to switch ? (want to also use French) is it possible ? I know google speech can take a langage field on `createRecognizeStream` but I didn't see any option to set that in Sonus

**Comment by jaumard** [comment]
Never mind :D found it :)
